aws/service.go

```diff
@@ -1027,6 +1027,14 @@ func S3ControlClient(ctx context.Context, d *plugin.QueryData, region string) (*
 	return s3control.NewFromConfig(*cfg), nil
 }
 
+func S3ControlMultiRegionAccessClient(ctx context.Context, d *plugin.QueryData, region string) (*s3control.Client, error) {
```

Comment:
We should add the comment you included below above this function, so it's clear why we need a separate client function

Reply:
Added comment